Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -205,21 +240,23 @@ - (void)loadInterstitialAdForParameters:(id<MAAdapterResponseParameters>)paramet
     NSString *placementIdentifier = parameters.thirdPartyAdPlacementIdentifier;
     [self log: @"Loading %@interstitial ad for placement: %@...", ( isBiddingAd ? @"bidding " : @"" ), placementIdentifier];
     
-    if ( ![VungleAds isInitialized] )
-    {
-        [self log: @"Vungle SDK not successfully initialized: failing interstitial ad load..."];
-        [delegate didFailToLoadInterstitialAdWithError: MAAdapterError.notInitialized];
-        
-        return;
-    }
-    
-    [self updateUserPrivacySettingsForParameters: parameters];
-    
-    self.interstitialAd = [[VungleInterstitial alloc] initWithPlacementId: placementIdentifier];
-    self.interstitialAdDelegate = [[ALVungleMediationAdapterInterstitialAdDelegate alloc] initWithParentAdapter: self andNotify: delegate];
-    self.interstitialAd.delegate = self.interstitialAdDelegate;
-    
-    [self.interstitialAd load: bidResponse];
+    [self initVungleSDK:parameters completionHandler:^(MAAdapterInitializationStatus initializationStatus, NSString * _Nullable errorMessage) {
+        if ( initializationStatus == MAAdapterInitializationStatusInitializedFailure ) 
+        {
+            [self log: @"Vungle SDK not successfully initialized: failing interstitial ad load..."];
```

Comment:
The error message has the error code and description. Can it be added in the log message?